ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/gui/PlayGamePanel.java

```diff
@@ -229,6 +231,16 @@ private JMenu createMenuBar() {
         return mOptionsMenu;
     }
 
+    private void resizeElements(int width, int height){
+        for (BoardPanel board : mGameBoards) {
+            board.rescaleBoard(width, height);
+        }
+
+        for (JailPanel jailPanel : mJails) {
```

Comment:
Same here:
`Stream.of(mJails).forEach(jail -> jail.rescaleBoard(width, height));`